src/fdc3/dotnet/DesktopAgent/src/DesktopAgent/DesktopAgent.csproj

```diff
@@ -1,21 +1,41 @@
-<Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
-    <ImplicitUsings>enable</ImplicitUsings>
-    <Nullable>enable</Nullable>
-    <AssemblyName>MorganStanley.ComposeUI.Fdc3.$(MSBuildProjectName)</AssemblyName>
-    <RootNamespace>MorganStanley.ComposeUI.Fdc3.$(MSBuildProjectName.Replace(" ", "_"))</RootNamespace>
-  </PropertyGroup>
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<TargetFramework>net6.0</TargetFramework>
+		<ImplicitUsings>enable</ImplicitUsings>
+		<Nullable>enable</Nullable>
+		<AssemblyName>MorganStanley.ComposeUI.Fdc3.$(MSBuildProjectName)</AssemblyName>
+		<RootNamespace>MorganStanley.ComposeUI.Fdc3.$(MSBuildProjectName.Replace(" ", "_"))</RootNamespace>
+	</PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="6.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Hosting" Version="6.0.1" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="6.0.4" />
-    <PackageReference Include="MorganStanley.Fdc3" Version="2.0.0-alpha.5" />
-  </ItemGroup>
+	<ItemGroup>
+		<PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="6.0.0" />
+		<PackageReference Include="Microsoft.Extensions.Hosting" Version="6.0.1" />
+		<PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="6.0.4" />
+		<PackageReference Include="MorganStanley.Fdc3" Version="2.0.0-alpha.5" />
+	</ItemGroup>
 
-  <ItemGroup>
-    <ProjectReference Include="..\..\..\..\..\messaging\dotnet\src\Core\MorganStanley.ComposeUI.Messaging.Core.csproj" />
-  </ItemGroup>
+	<ItemGroup>
+		<ProjectReference Include="..\..\..\..\..\messaging\dotnet\src\Core\MorganStanley.ComposeUI.Messaging.Core.csproj" />
+	</ItemGroup>
+
+	<PropertyGroup>
+		<Fdc3BundleOutput>$(ComposeUIRepositoryRoot)\src\fdc3\js\composeui-fdc3\dist\fdc3-iife-bundle.js</Fdc3BundleOutput>
```

Comment:
I'm not sure it's a good idea to move the embedded JS bundle to the Fdc3 dotnet project, it doesn't belong there. The shell is what wires these together, but the desktop agent implementation for .NET doesn't depend on the JS package. I'd move the resource back to the shell project and make it optional to 
- add the FDC3 backend
- add the JS bundle to preload scripts